core/templates/components/question-directives/question-player/skill-mastery-modal.controller.spec.ts

```diff
@@ -44,13 +44,14 @@ describe('Skill Mastery Modal Controller', function() {
     });
   }));
 
-  it('should check properties set after controller is initialized', function() {
-    expect($scope.userIsLoggedIn).toEqual(userIsLoggedIn);
-    expect($scope.skillId).toEqual(skillId);
-    expect($scope.masteryChange).toEqual('mastery change');
-  });
+  it('should evaluate $scope properties after controller is initialized',
+    function() {
+      expect($scope.userIsLoggedIn).toEqual(userIsLoggedIn);
+      expect($scope.skillId).toEqual(skillId);
+      expect($scope.masteryChange).toEqual('mastery change');
+    });
 
-  it('should open concept card', function() {
+  it('should open concept card successfully', function() {
```

Comment:
Same thing -- "it should [do X] [under condition Y]". Here and elsewhere. You also don't really need "successfully" in general; that is implied.

Reply:
Ditto elsewhere -- please check your description strings for this.

Reply:
Done.